spec/controllers/api/v1/notes_controller_spec.rb

```diff
@@ -104,4 +104,66 @@
       end
     end
   end
+
+  describe 'POST #create' do
+    let(:note_params) { { title: 'New Note', note_type: 'review', content: 'Sample content' } }
+
+    context 'when user is logged in' do
+      include_context 'with authenticated user'
+
+      context 'when parameters are valid' do
+        before { post :create, params: { note: note_params } }
+
+        it 'creates a new note' do
+          expect(Note.count).to eq(1)
+        end
```

Comment:
Excelente! Otra forma de encarar este caso a testear es probar que se incremente el total en una X cantidad, algo asi:

```suggestion
        it 'creates a new note' do
          expect { post :create, params: { note: note_params } } .to change(Note, :count).by(1)
        end
```
La sintaxis en el expect cambia un poco porque estamos pasándole un bloque de código en lugar de un parámetro, pero en resumen se lee como "espero que crear una nota nueva me incremente en uno la cantidad de notas"

Reply:
al final decidi mantenerme con la sintaxis actual ya que al usar el post al principio del contexto, no funcionaba como esperaba tener 2 post